main/saturn-node.js

```diff
@@ -9,8 +8,7 @@ const { fetch } = require('undici')
 const fs = require('node:fs/promises')
 const path = require('path')
 const { setTimeout } = require('timers/promises')
-
-const configStore = new Store()
+const { getFilAddress, setFilAddress } = require('./station-config')
```

Comment:
Is this backwards compatible? When I upgrade my current Station that already has my FIL address configured, will the new version find that FIL address?

Reply:
As long as they keys stay the same, this is backwards compatible. Since the store from `station-config` doesn't have a `.name` configured, and the store from `saturn-node` didn't have one either, they were effectively the same store.